<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsLicenceManagerDlg</class>
 <widget class="QDialog" name="CwmsLicenceManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WMS Licence Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="m_pqlArticleType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(232, 232, 232, 255));</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Lizenzverwaltung&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Aktive Sitzungen</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="m_pqlvSessions">
         <column>
          <property name="text">
           <string>Sitzungs Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Benutzer Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Login Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Login Zeitpunkt</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="m_pqpbFinishSession">
           <property name="toolTip">
            <string>Sitzung beenden</string>
           </property>
           <property name="statusTip">
            <string>Sitzung beenden</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbStopAllSessions">
           <property name="toolTip">
            <string>Alle Sitzungen beenden</string>
           </property>
           <property name="statusTip">
            <string>Alle Sitzungen beenden</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/DeleteFolder.png</normaloff>:/icons/DeleteFolder.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>181</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lizenzen</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>Lizenznehmer</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_pqleLicencee"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>Software</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_pqleSoftwareName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel4">
           <property name="text">
            <string>Lizenzschlüssel</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="m_pqleLicenceKey"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_pqpbAdd">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="m_pqlvLicences">
         <column>
          <property name="text">
           <string>Lizenznehmer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Lizenzschlüssel</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Software</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Anzahl der Lizenzen</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ablaufdatum</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel1">
           <property name="text">
            <string>Gültige Lizenzen</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_pqleValidLicences">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>130</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbDeleteLicence">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbRefresh">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/refresh24.png</normaloff>:/icons/refresh24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbQuit">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../WMSadmin/resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbAdd</sender>
   <signal>clicked()</signal>
   <receiver>CwmsLicenceManagerDlg</receiver>
   <slot>AddClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbFinishSession</sender>
   <signal>clicked()</signal>
   <receiver>CwmsLicenceManagerDlg</receiver>
   <slot>FinishSessionClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbQuit</sender>
   <signal>clicked()</signal>
   <receiver>CwmsLicenceManagerDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRefresh</sender>
   <signal>clicked()</signal>
   <receiver>CwmsLicenceManagerDlg</receiver>
   <slot>RefreshClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbDeleteLicence</sender>
   <signal>clicked()</signal>
   <receiver>CwmsLicenceManagerDlg</receiver>
   <slot>DeleteLicenceClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbStopAllSessions</sender>
   <signal>clicked()</signal>
   <receiver>CwmsLicenceManagerDlg</receiver>
   <slot>StopAllSessionsClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>StopAllSessionsClickedSlot()</slot>
 </slots>
</ui>
